Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -17,6 +17,65 @@ This integration was integrated and tested with [Security & Compliance Center](h
 
 ## Permissions in the Security & Compliance Center
 
+### Overview
+In response to Microsoft's deprecation of the App ID, the following changes to existing app registration in Azure are required:
```

Comment:
maybe this needs a more zoom out overview?
somthing like, this section provides information abut updating configurations that used xsoar(?)app ID that was deprecated by Microsoft in <date>